README.md

```diff
@@ -30,4 +30,6 @@ Give a star if you like this project!
 - [@bolah2009](http://github.com/bolah2009)
 
 ## License
+
 AdejamTech License
+
```

Comment:
Let's add a live link. To do that we can set up GitHub Pages, see https://pages.github.com/
Or alternatively, we can also use https://raw.githack.com/, below is an example of using Githack.
1. Go to the HTML file on GitHub and copy the link
2. Go to https://raw.githack.com/ and paste the link
3. Two links are generated, one with the lastest commit and one with the branch from where the HTML link is copied
4. You can use either of this link. Using the second one makes sure that lastest changes are updated in the page while the first one doesn't change even after updating git with new changes, this means you have to generate a link for every changes.

![how-to-add-live-link](https://user-images.githubusercontent.com/36057474/77068748-60c35f80-69e7-11ea-99c3-778b773cb548.gif)